public/achievementList.php

```diff
@@ -216,8 +216,13 @@
                     echo "</td>";
                 }
             } else {
+                $renderAwardedDate = "Unknown";
```

Comment:
A blank awarded date implies the achievement was not earned. Recommend using blank instead of Unknown.

Although, I can't get these to show up any more. The other changes seem to have made it impossible to list awarded dates for unearned achievements.

Reply:
I do have a tiny handful in my database where the `Date` value is set to `NULL`, so at some point it must have been possible (or just something is wrong with the DB dump):

![Screenshot 2023-07-02 at 10 51 36 AM](https://github.com/RetroAchievements/RAWeb/assets/3984985/11d7b6b7-c93b-4aa5-95a6-603e3cc92ce0)

Reply:
I think you're counter-point has swayed my decision.

I can't make the list show unearned achievements, but I can make it show earned achievements with a null date.

Reply:
Got it- just to confirm before I make a commit, should I set this string back to display "Unknown"?

Reply:
Yes, please.

Reply:
No problem, I've set this to display "Unknown" again in https://github.com/RetroAchievements/RAWeb/pull/1631/commits/c18a9897267ad67148c86a351ebe94f66edfaeed.